ee/control/consumers/notificationconsumer/notify_consumer.go

```diff
@@ -220,7 +220,6 @@ func (nc *NotificationConsumer) cleanup() {
 
 	// Delete all old keys
 	if err := nc.store.Delete(keysToDelete...); err != nil {
-		// Log, but don't error, since we might be able to delete some others
 		level.Error(nc.logger).Log("msg", "could not delete old notifications from bucket", "err", err)
```

Comment:
```suggestion
		level.Debug(nc.logger).Log("msg", "could not delete old notifications from bucket", "err", err)
```
(I know it's old code, but here we are)